Makefile

```diff
@@ -3,24 +3,23 @@ all: develop
 
 PIP := python -m pip --disable-pip-version-check
 WEBPACK := yarn build-acceptance
-POSTGRES_CONTAINER := sentry_postgres
 
 freeze-requirements:
 	@python3 -S -m tools.freeze_requirements
 
 bootstrap \
 develop \
+install-js-dev \
+install-py-dev \
+apply-migrations:
+	@devenv sync
```

Comment:
I kinda hate hiding the commands like this -- I'd rather unhide this

Reply:
I think one problem with this is if someone runs `make install-js-dev install-py-dev` it'll run `devenv sync` twice

probably need a separate PHONY target for `devenv-sync` that these all depend on

Reply:
oh wow, that's really smart..!